src/pickup.h

```diff
@@ -0,0 +1,37 @@
+#ifndef _PICKUP_H
+#define _PICKUP_H
+
+#include <unordered_map>
+
+#include "entity.h"
+
+namespace pickupNS {
+	const enum PICKUP_TYPE { MAGNET, INVERT };
```

Comment:
Yea I know theres a map to convert it but it still means when creating a new effect we need to add it to the json, the enum and the unordered_map